docs/sphinx/source/whatsnew/v0.9.0.rst

```diff
@@ -65,6 +65,22 @@ Deprecations
   * ``ModelChain.total_irrad``
   * ``ModelChain.tracking``
 
+* The following attributes of :py:class:`pvlib.pvsystem.PVSystem` and
+  :py:class:`pvlib.tracking.SingleAxisTracker` have been deprecated in
+  favor of the corresponding :py:class:`pvlib.pvsystem.Array` attributes:
+
+  * ``PVSystem.module_parameters``
+  * ``PVSystem.module``
+  * ``PVSystem.module_type``
+  * ``PVSystem.albedo``
+  * ``PVSystem.temperature_model_parameters``
+  * ``PVSystem.surface_tilt``
+  * ``PVSystem.surface_azimuth``
+  * ``PVSystem.racking_model``
+  * ``PVSystem.modules_per_string``
+  * ``PVSystem.strings_per_inverter``
```

Comment:
I like the alphabetization of the list preceding this one and would prefer to see that here too.